src/main/webapp/resources/js/pages/admin/components/AdminSideMenu.jsx

```diff
@@ -33,19 +33,19 @@ export default function AdminHeader() {
                 <img style={{ height: 64, width: 200 }} src={setBaseUrl("/resources/img/irida_logo_dark.svg")}/>
               </Link>
               <Menu className={"t-admin-side-menu"} style={{ height: '100vh' }} theme={"dark"} mode={"inline"}
-                    selectedKeys={[keyname ? keyname[1] : ADMIN.STATISTICS]}>
+                    selectedKeys={[keyname ? keyname[0].split('/')[0] : ADMIN.STATISTICS]}>
```

Comment:
Can you create a function that does this instead of doing it inline and add a comment to the function as to why this needs to be done

`keyname[0].split('/')[0]`